src/types/remark.ts

```diff
@@ -0,0 +1,23 @@
+import type { Root } from 'mdast'
```

Comment:
Added this type file to avoid doing `$TSFixMe` for the `additionalRemarkPlugins` option in `docs/server`.

Reply:
`unified` exports a `Pluggable` type, which we might be able to use. ref: https://github.com/hashicorp/next-mdx-remote/blob/main/src/types.ts#L12

Reply:
✅  Great call - updated to use `Plugin` & `Pluggable` from `unified`, deleted this file 👍